Gemfile.lock

```diff
@@ -1,203 +0,0 @@
-GEM
```

Comment:
@camlow325 We decided to break Puppet Labs tradition and commit this file alongside Gemfile per the recommended Bundler practice (see http://bundler.io/).